cv-basics/OpenCV_Overview.ipynb

```diff
@@ -0,0 +1,939 @@
+{
```

Comment:
<p>Please read the documentation of cv2.waitkey() 0 means forever</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SRA-VJTI/Intro_to_CV/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
change encorporated, thanks